notifier/src/main/kotlin/modules/JiraNotifier.kt

```diff
@@ -47,6 +48,26 @@ class JiraNotifier(
         restClient.issueClient.addComment(issue.commentsUri, Comment.valueOf(comment)).claim()
     }
 
+    /**
+     * Change the [state] of the given issue specified by the [issueKey].
+     */
+    fun changeState(issueKey: String, state: String): Result<Boolean> {
+        val issue = restClient.issueClient.getIssue(issueKey).claim()
+        val possibleTransitions = restClient.issueClient.getTransitions(issue).claim()
+
+        val transition = possibleTransitions.singleOrNull { it.name == state } ?: return Result.failure(
+            IllegalArgumentException("The transition to the state '$state' is not possible.")
+        )
+
+        return runCatching {
+            restClient.issueClient.transition(issue, TransitionInput(transition.id)).claim()
+            true
```

Comment:
There is no advantage of using `Result<Boolean>` as a return value. The success of the creation is already shown by `Result` itself.
It should be possible to use `Result<Void>` and to remote the `true` as a return value.

Reply:
If `changeState` is not required to return a `Result` object, let it simply return `Boolean`, drop the `true` here, and use `.isSuccess` in line 68.

Reply:
I prefer using `Result<Void>` otherwise https://github.com/oss-review-toolkit/ort/blob/54c206ba4629849ebe542c7cfdd74d0d49698668/notifier/src/main/kotlin/modules/JiraNotifier.kt#L58-L59 would crash the notifier if a transition is not possible for one issue.